src/components/templates/Audience/Audience.scss

```diff
@@ -148,28 +155,44 @@ $light: #ffffff;
   .audience {
     display: flex;
     flex: 1;
-    padding-left: 30px;
-    padding-right: 30px;
-    padding-top: 30px;
-    padding-bottom: 30px;
     flex-direction: column;
     justify-content: space-between;
+    position: relative;
+    width: 100%;
+    height: fit-content;
+    overflow: auto;
 
     .seat-row {
       display: flex;
-      justify-content: space-between;
+      justify-content: center;
     }
 
-    .seat {
+    .seat,
+    .not-seat,
+    .profile-avatar {
       display: flex;
-      border-radius: 100%;
-      width: 4vh;
-      height: 4vh;
       justify-content: center;
-      align-content: center;
-      background-color: rgba(255, 255, 255, 0.2);
-      color: white;
+      align-items: center;
+
+      width: var(--seat-size);
+      height: var(--seat-size);
+      min-width: var(--seat-size-min);
+      min-height: var(--seat-size-min);
+
+      font-size: calc(min(#{$seat-size}, #{$seat-size-min}) * 0.75);
+
+      border-radius: 100%;
+
+      color: $black;
       user-select: none;
+    }
+
+    .seat {
+      background-color: rgba(255, 255, 255, 0.2);
+
+      .add-participant-button {
+        z-index: z(audience-participant-add-button);
```

Comment:
[note] I tried removing this + tried a bunch of alternatives, still had issues, not sure why. tl;dr: Worth keeping it for now.